components/x-teaser/src/Video.jsx

```diff
@@ -20,12 +22,14 @@ const Embed = (props) => (
 			data-o-video-optimumvideowidth="640"
 			data-o-video-autorender="true"
 			data-o-video-playsinline="true"
+			data-o-video-show-guidance={showGuidance}
```

Comment:
Maybe we should make `showGuidance` a string to make it consistent with the other data attributes.